test/compress/functions.js

```diff
@@ -1456,3 +1456,85 @@ issue_2630_5: {
     }
     expect_stdout: "155"
 }
+
+issue_2647_1: {
+    options = {
+        inline: true,
+        reduce_vars: true,
+        side_effects: true,
+        unused: true,
+    }
+    input: {
+        (function(n, o = "FAIL") {
+            console.log(n);
+        })("PASS");
```

Comment:
Add a test case with no call arguments to see what happens with the default parameter.

Reply:
Isn't the case right below this one already done that? :sweat_smile:

Reply:
The test below has an argument "FAIL". Please humour me with a no call argument test case.

Reply:
Done in https://github.com/mishoo/UglifyJS2/pull/2648/commits/46067059600c611efdf5383631e00cd91670b2b6

Reply:
Thanks